README.md

```diff
@@ -8,7 +8,7 @@
 
 ## Why MemoWise?
 
-**MemoWise is the wise choice for memoization in Ruby.**
+**{MemoWise} is the wise choice for memoization in Ruby.**
```

Comment:
I'm not sure I understand this... won't this look weird in the GitHub `README.md` view?

Reply:
Ah sure -- I can move the link elsewhere